composer.json

```diff
@@ -15,6 +15,7 @@
   ],
   "require": {
     "drupal/context": "^4 || ^5@RC",
+    "drupal/core": "^10.2",
```

Comment:
Don't think is needed especially with a long term switch to drupal.org https://www.drupal.org/docs/develop/using-composer/add-a-composerjson-file#core-compatibility.